paging/paging-common/src/test/kotlin/androidx/paging/PagingDataDifferTest.kt

```diff
@@ -1577,10 +1922,28 @@ class PagingDataDifferTest(
                     ).also { pagingSources.add(it) }
                 }
             ),
-        block: (SimpleDiffer, TestDispatcher, MutableList<TestPagingSource>) -> Unit
+        block: (
+            SimpleDiffer,
+            TestDispatcher,
+            MutableList<TestPagingSource>,
+            MutableList<TrackableUiReceiverWrapper>,
+            MutableList<TrackableHintReceiverWrapper>
```

Comment:
This could be `hintReceivers: List<TrackableHintReceiverWrapper>` to give autocomplete to Android Studio on what these variables should be named.